frontend/src/app/containers/UserDetailPage/index.tsx

```diff
@@ -40,6 +42,69 @@ export function UserDetailPage(props: Props) {
 
   if (user === null) return <></>;
 
+  const addBanButton = () => {
+    if (user === null) return <></>;
+    if (currentUser === null) return <></>;
+    if (user.id === currentUser.id) {
+      return <></>;
+    } else {
+      return (
+        <BanButton
+          userId={user.id}
+          isBanned={user.status === 100}
+          onBanStatusChanged={function () {
+            toast.success('User status changed successfully');
+            history.push('.');
+          }}
+        ></BanButton>
+      );
+    }
+  };
+
+  const addUpdateButton = () => {
```

Comment:
try to write this in an separate file, you can delete the old ban button if needed